extra/hip_wrapper.py

```diff
@@ -681,5 +680,9 @@ def hiprtcGetCode(prog):
     status = _libhiprtc.hiprtcGetCode(prog, e_code)
     hipCheckStatus(status)
     return e_code
+
+  def language(**kwargs):
```

Comment:
This shouldn't be in here, it's just hiding complexity.

Reply:
I tried to minimize the impact on ops_hips.py, do you have any suggestions?

Reply:
Just put it in ops_hip. If it's required complexity that's okay, but it shouldn't hide

Reply:
but then ops_hip will need another "if HIP_CPU"

Reply:
unfortunately HIP-CPU is a bit of a mess and lacks a few APIs (e.g. capture), maybe HIP_CPU should become a Device? so tests can filtered it out.

Reply:
If it's a different Device there's no point in doing this, what's being tested?

Reply:
"but then ops_hip will need another "if HIP_CPU""

and...? if you hide complexity, you still have the complexity, it's just elsewhere. surfaced complexity > hidden complexity.

Reply:
> "but then ops_hip will need another "if HIP_CPU""
> 
> and...? if you hide complexity, you still have the complexity, it's just elsewhere. surfaced complexity > hidden complexity.

It's an interface provided by a hip cpu wrapper, like all those hip_ functions, implemented by both wrappers. Should I move all implementations to ops_hip?

Reply:
> If it's a different Device there's no point in doing this, what's being tested?

test_ops is being tested